client/components/bcat-form-sections/general-project-info.tsx

```diff
@@ -46,7 +46,7 @@ export const GeneralProjectInfo: React.FC = () => {
                 options={[
                   {
                     label:
-                      'Facility master plan (small facilities only, maximum BCAAP contribution is $25,000)',
+                      'Facility master plan (small facilities only, maximum BCAT contribution is $25,000)',
```

Comment:
We don't have facility master plan

Reply:
I assume this whole radio will change/ be removed, I'm just not sure what the options would be at the moment